FrontEnd/src/components/SignUp/components/signup/signup-form/ResendActivationFormContent.js

```diff
@@ -0,0 +1,90 @@
+import { useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate  } from 'react-router-dom';
+
+import axios from 'axios';
+import styles from './ResendActivationFormContent.module.css';
+
+export function ResendActivationFormContentComponent(props) {
+  const navigate = useNavigate();
+
+  const errorMessageTemplates = {
+    required: 'Обов’язкове поле',
+    email: 'Email не відповідає вимогам',
+  };
+
+  const emailPattern = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+
+  const {
+    register,
+    handleSubmit,
+    getValues,
+    formState: { errors, isValid },
+  } = useForm({
+    mode: 'all',
+  });
+
+  const { setIsValid } = props;
+
+  useEffect(() => {
+    const formIsValid = isValid;
+    setIsValid(formIsValid);
+  }, [isValid, setIsValid]);
+
+  const onSubmit = () => {
+    const dataToSend = {
+      email: getValues('email'),
+    };
+
+    axios({
+      method: 'post',
+      url: `${process.env.REACT_APP_BASE_API_URL}/api/auth/users/resend_activation/`,
+      withCredentials: false,
+      data: dataToSend
+    }).then(res => console.log(res.data)).catch(error => console.log(error));
+    console.log(process.env.REACT_APP_BASE_API_URL);
+    navigate('/login');
```

Comment:
I think we should navigate users only when activation is successful and show error when smth went wrong

Reply:
Fixed.
What about error, It will probably be better to do this in a new task because I need to come up with a design for the error?

Reply:
I do not think that we have a design for every second error state. I would recommend using existing toast alerts

Reply:
Fixed